README.md

```diff
@@ -1,5 +1,134 @@
 # Joy in OCaml
 
-More information coming soon.
+## Table of Contents
+- [Project Overview](#project-overview)
+- [Project Benefits](#project-benefits)
+- [Installation](#installation)
+- [Getting Help](#getting-help)
+- [Other wonderful Shapes](#other-wonderful-shapes)
+- [Contribution](#contribution)
+- [Acknowlegments](#acknowlegments)
+
+## Project Overview
+This library is like a toolbox for creating cool shapes and patterns using OCaml. It takes inspiration from [joy](https://github.com/fossunited/joy). Whether you're an artist or a coder, you can use it to make interesting art with code.
+
+## Project Benefits
+
+Here's how this project can make your life easier and more creative:
+
+1. **Get Creative with Geometry:** This project simplifies geometric creative coding in OCaml, helping you explore the exciting world where programming meets art.
+
+2. **User-Friendly Tools:** We provide an easy-to-use API, so whether you're a newbie or a pro, you can dive into creative coding without confusion.
+
+3. **Basic Shapes at Your Fingertips:** Use simple shapes as your creative building blocks, making it a breeze to start your artistic coding journey.
+
+4. **Mix and Match with Ease:** Combine, transform, and build complex shapes effortlessly.
+
+
+## Installation
```

Comment:
This is quite useful, especially considering this library is aimed to be easy to use for non-OCaml programmers. However, I'd prefer the Installation instructions and the contribution section moved to a separate file, `HACKING.md` or `CONTRIBUTING.md`. The new file can be referenced in the README.

Reply:
Thank you, @Sudha247! I'll start working on the changes right away.